spec/reek/smells/instance_variable_assumption_spec.rb

```diff
@@ -0,0 +1,184 @@
+require_relative '../../spec_helper'
+require_lib 'reek/smells/instance_variable_assumption'
+
+RSpec.describe Reek::Smells::InstanceVariableAssumption do
```

Comment:
All of these specs test assignment to ivars. I think the important part of this smell is detection of use, like:

```
  def foo
    @foo
  end
```